packages/components-js/tests/e2e/specs/banner.e2e.ts

```diff
@@ -12,18 +12,24 @@ import {
   setProperty,
   waitForStencilLifecycle,
 } from '../helpers';
-import type { BannerState } from '@porsche-design-system/components';
+import type { BannerState, BannerHeadingTag } from '@porsche-design-system/components';
 
 type InitOptions = {
   open: boolean;
   state?: BannerState;
   dismissButton?: boolean;
+  headingTag?: BannerHeadingTag;
 };
 
 const initBanner = (page: Page, opts: InitOptions): Promise<void> => {
-  const { open = false, state, dismissButton = true } = opts || {};
+  const { open = false, state, dismissButton = true, headingTag } = opts || {};
 
-  const attrs = [`open="${open}"`, state ? `state="${state}"` : '', `dismiss-button="${dismissButton}"`].join(' ');
+  const attrs = [
+    `open="${open}"`,
+    state ? `state="${state}"` : '',
+    `dismiss-button="${dismissButton}"`,
+    headingTag && `heading-tag="${headingTag}"`,
```

Comment:
Doesn't this produce undefined as prop on the `<p-banner>` when headingTag isn't used?